plugins/source/postgresql/resources/plugin/plugin_test.go

```diff
@@ -52,20 +53,78 @@ func getTestConnectionString() string {
 	return testConn
 }
 
-func createTestTable(ctx context.Context, conn *pgxpool.Pool, table *schema.Table) error {
+type pgTypeToValue struct {
+	typeName string
+	value    any
+}
+
+func getPgTypesToData() []pgTypeToValue {
+	var pgTypesToData = []pgTypeToValue{
+		{"bigint", 1},
+		{"bigserial", nil},
+		{"bit", "1"},
+		{"bit(5)", "11111"},
+		{"bit varying", "1"},
+		{"bit varying(5)", "11111"},
+		{"boolean", true},
+		{"box", "((1,2),(3,4))"},
+		{"bytea", []byte("test")},
+		{"character", "a"},
+		{"character(5)", "aaaaa"},
+		{"character varying", "a"},
+		{"character varying(5)", "aaaaa"},
+		{"cidr", "10.1.2.3/32"},
+		{"circle", "<(1,2),3>"},
+		{"date", "1999-01-08"},
+		{"double precision", 1.1},
+		{"inet", "192.168.0.1/24"},
+		{"integer", 1},
+		{"interval", "1-2"},
+		{"json", `{"a":1}`},
+		{"jsonb", `{"a":1}`},
+		{"line", "{1,2,3}"},
+		{"lseg", "[(1,2),(3,4)]"},
+		{"macaddr", "08:00:2b:01:02:03"},
+		{"macaddr8", "08:00:2b:01:02:03:04:05"},
+		{"money", "$1,000.00"},
+		{"path", `[(1,2),(3,4)]`},
+		{"point", "(1,2)"},
+		{"polygon", `((1,2),(3,4))`},
+		{"real", 1.1},
+		{"smallint", 1},
+		{"smallserial", nil},
+		{"serial", nil},
+		{"text", "test"},
+		{"time without time zone", "04:05:06.789"},
+		{"time(3)", "04:05:06.789"},
+		{"time(3) without time zone", "04:05:06.789"},
+		{"timestamp", "1999-01-08 04:05:06.789"},
+		{"timestamp without time zone", "1999-01-08 04:05:06.789"},
+		{"timestamp(3)", "1999-01-08 04:05:06.789"},
+		{"timestamp(3) without time zone", "1999-01-08 04:05:06.789"},
+		{"tsquery", "a & b"},
+		{"tsvector", "'a':1 'b':2"},
+		{"uuid", uuid.New().String()},
+		{"xml", "<a>1</a>"},
+	}
+
+	return pgTypesToData
+}
+
+func createTestTable(ctx context.Context, conn *pgxpool.Pool, tableName string) error {
 	var sb strings.Builder
 	sb.WriteString("CREATE TABLE ")
-	sb.WriteString(table.Name)
+	sb.WriteString(tableName)
 	sb.WriteString(" (")
-	c := client.Client{}
-	for i, col := range table.Columns {
-		sb.WriteString(col.Name)
+	columns := getPgTypesToData()
+	for i, col := range columns {
+		sb.WriteString(pgx.Identifier{col.typeName + "_type"}.Sanitize())
```

Comment:
Why not use column names from DB?

Reply:
will create clash on several columns with the same type

Reply:
Can you please explain your comment? We don't get the column names from the DB. `columns` have test data generated here https://github.com/cloudquery/cloudquery/pull/8926/files#diff-715416571f0e3a5047731c92c67565cf315764ec01e5b2d59969b00bc99a0b70R61

Reply:
Ah, I reviewed and didn't notice that this is a test.